connect_four.py

```diff
@@ -186,7 +184,8 @@ def new_turn(self, player: str, color: str) -> None:
             self.winner = player
             self.play = False
         
-        if self.board.board_full:
+        #check if board is full, possible draw scenario
+        if self.board.is_board_full:
```

Comment:
Make sure you remember the function call like:

self.board.is_board_full() or else the truthiness of a function reference is always true, which will give you a bug.

Reply:
Dang, sorry I rushed through it.